feeds/trilliumtransit.com.dmfr.json

```diff
@@ -3139,16 +3181,39 @@
         }
       ]
     },
+    {
+      "id": "f-carts~ak~us",
+      "spec": "gtfs",
+      "urls": {
+        "static_current": "https://data.trilliumtransit.com/gtfs/carts-ak-us/carts-ak-us.zip"
+      }
+    },
     {
       "id": "f-cata~pa~us",
       "spec": "gtfs",
       "urls": {
-        "static_historic": [
-          "https://data.trilliumtransit.com/gtfs/cata-pa-us/cata-pa-us--imap.zip"
-        ]
-      },
-      "tags": {
-        "status": "outdated"
+        "static_current": "https://data.trilliumtransit.com/gtfs/cata-pa-us/cata-pa-us.zip"
+      }
+    },
+    {
+      "id": "f-catholiccommunity~wa~us~king",
+      "spec": "gtfs",
+      "urls": {
```

Comment:
if these 3x catholiccommunity feeds should be grouped together, let's create a single operator and associate it with all 3 feeds